src/types/schema.ts

```diff
@@ -1,65 +1,105 @@
 import { UUID } from 'crypto';
 
-export interface CaseListing {
-  id: UUID;
-  title: string;
-  summary: string;
-  country: string;
-  client_location: string;
-  legal_server_id: number;
-  hours_per_month: number;
-  num_months: number;
-  languages: string[];
-  is_remote: boolean;
-  in_court: boolean;
-  needs_attorney: boolean;
-  needs_interpreter: boolean;
-  upcoming_date: string;
-  relief_sought: string;
-  active: boolean;
+// table field types
+export type ImmigrationLawExperienceEnum = 'LOW' | 'MEDIUM' | 'HIGH';
+
+export type AdjudicatingAgencyEnum =
```

Comment:
Related to your message on Slack, what are your thoughts on making adjudicating agency a string instead of an enum? I think that might allow for more flexibility, in case they add another potential agency afterward

Reply:
Yeah I think that makes sense given legal server data may not be standardized to fit into the enum

Reply:
We should also note that we will be pulling `experience_needed` from legal server as well, and we may want to ask them about the format it'll be in